### ISSUE TITLE
CMake searches for LAME pkgconfig file that does not exist

### ISSUE THREAD

**Issue by Be-ing** [tenacityteam/tenacity-legacy#225]
- [x] I have read the specified guidelines for issues

**Describe the bug**
When building with system libraries using `-Dlib_preference=system -Dobey_system_dependencies=ON`, configuration fails because CMake [searches for a pkgconfig file for LAME](https://github.com/tenacityteam/tenacity/blob/49015e77d0cf39430d180b88f29a68e036235e81/cmake-proxies/CMakeLists.txt#L85), but LAME has no pkgconfig file:

```
❯ rpm -ql lame-devel
/usr/include/lame
/usr/include/lame.h
/usr/include/lame/lame.h
/usr/lib64/libmp3lame.so
/usr/share/doc/lame-devel
/usr/share/doc/lame-devel/API
/usr/share/doc/lame-devel/HACKING
/usr/share/doc/lame-devel/STYLEGUIDE
```

I downloaded the LAME source code and confirmed there is indeed no pkgconfig file for LAME; this is not a bug in Fedora's package:
```
~/sw/lame-3.100 
❯ find . -name \*.pc

~/sw/lame-3.100 
❯ 
```

**Additional information (please complete the following information):**
 - OS: Fedora
 - Version: 34

**Additional context**
Originally reported by @dvzrv at https://github.com/audacity/audacity/issues/521

**Comment by Be-ing** [comment]
We could copy the [Findmp3lame.cmake module from Mixxx](https://github.com/mixxxdj/mixxx/blob/main/cmake/modules/Findmp3lame.cmake). It is licensed GPLv2 or later.

**Comment by GriffintheFolf** [comment]
Wouldn't we also need to add an acknowledgement somewhere for using this file, to comply with the GPLv2? Probably under `3RDPARTY.txt` would seem most appropriate.

**Comment by Be-ing** [comment]
There's an acknowledgement at the top of the file, but I suppose it wouldn't hurt to acknowledge it elsewhere as well :shrug:

**Comment by GriffintheFolf** [comment]
I didn't think of that, whoops.

**Comment by asperkraken** [comment]
Note: I believe i got the same error message on my first build as well.
The second time i built it, same programs and all, same destinations, i had ZERO errors.
Not sure what does this but it seemed to happen whenever my compiler was not configured to CMake prior to the build.
Try building a second time, following the instructions in BUILDING.MD
My second build I did an MP3 export with zero issue.

Note: current Audacity (non-forked) builds have Lame included. Perhaps the secondary install caused an error?

**Comment by Be-ing** [comment]
Fixed in #228
